### ISSUE TITLE
Update to Android NDK r26

### ISSUE THREAD

**Issue by levinli303** [actions/runner-images#8327]
### Tool name

Android NDK

### Tool license

Apache

### Add or update?

- [ ] Add
- [X] Update

### Desired version

26.0.10792818

### Platforms where you need the tool

- [X] Azure DevOps
- [X] GitHub Actions

### Runner images where you need the tool

- [X] Ubuntu 20.04
- [X] Ubuntu 22.04
- [X] macOS 11
- [X] macOS 12
- [X] macOS 13
- [X] Windows Server 2019
- [X] Windows Server 2022

**Comment by erik-bershel** [comment]
@levinli303 the latest version of android ndk is expected with the next release of images.

**Comment by erik-bershel** [comment]
@levinli303 PRs merged, awaiting deployment.

**Comment by Javernaut** [comment]
The r26b version of the NDK is already available and I created a separate issue #8517 which was closed as a duplicate. May I ask why? Will the NDK r26b be available after #8327 instead?

**Comment by mikhailkoliada** [comment]
We install the latest available version, be it b or the newest one, so it does not matter :)

**Comment by mikhailkoliada** [comment]
Deployed
